src/component/fiber.ts

```diff
@@ -222,6 +222,7 @@ export class Fiber {
     if (this.target) {
       this.target.appendChild(this.component.el!);
       inDOM = document.body.contains(this.target);
+      this.component.env.qweb.trigger('dom-appended');
```

Comment:
we should add a reference page in the `doc/architecture` to document all _internal_ events used by owl.